code/modules/power/supermatter/supermatter.dm

```diff
@@ -476,6 +476,8 @@
 /obj/machinery/power/supermatter_shard/attack_paw(mob/user)
 	return attack_hand(user)
 
+/obj/machinery/power/supermatter_shard/attack_animal(mob/living/simple_animal/user)
+	return attack_hand(user)
```

Comment:
1. ```attack_hand()``` is not built in.
2. ```attack_X()``` calling ```attack_Y()``` is a fairly common pattern in the code already, it's fine.
3. What are you smoking.

@Cyberboss